config/config.json.example

```diff
@@ -1,7 +1,5 @@
 {
   "stops": ["Studio Arts Building", "Fine Arts Center", "Haigis Mall"],
-  "mail_settings": {"to": "transit-it@admin.umass.edu",
-                   "from": "transit-it@admin.umass.edu",
-                   "subject": "PVTA realtime feed error"},
+  "github_token": "9fe5e3055d8f4e2fd70180cf1fc1616db6b2b948",
```

Comment:
This is not a real GH access token, just a random hex string from https://random.org.